automatic/prtgdesktop/ReadMe.md

```diff
@@ -0,0 +1,3 @@
+﻿# PRTG Desktop
+
+PRTG Desktop is an alternative interface that you can use to connect to a PRTG core server or a PRTG hosted by Paessler instance to configure your setup, view monitoring results, and keep an eye on your network.
```

Comment:
Try to stick to the format of the description described in [$1.2.3](https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#123-informative-description) as much as possible. The website lists features which you can list here.